plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/_core.py

```diff
@@ -218,7 +219,18 @@ def sweep_over_batches(
             )
             self.job_idx += len(rets)
             for idx in range(len(batch)):
-                val = rets[idx].return_value
+                val: Any = rets[idx].return_value
+                # Ax expects a measured value (int or float), which can optionally
+                # be given in a tuple along with the error of that measurement
+                assert isinstance(val, (int, float, tuple))
+                # is_noisy specifies how Ax should behave when not given an error value.
+                # if true (default), the error of each measurement is inferred by Ax.
+                # if false, the error of each measurement is set to 0.
+                if isinstance(val, (int, float)):
+                    if not self.is_noisy:
+                        val = (val, 0)
+                    else:
+                        val = (val, None)
```

Comment:
This seems a bit clearer:
```suggestion
                    if self.is_noisy:
                        val = (val, None) # specify unknown noise
                    else:
                        val = (val, 0) # specify no noise
```